lighthouse-core/gather/gatherers/seo/tap-targets.js

```diff
@@ -257,10 +230,18 @@ function gatherTapTargets() {
   /** @type {LH.Artifacts.TapTarget[]} */
   const targets = [];
 
+  // Capture element positions relative to the top of the page
+  window.scrollTo(0, 0);
```

Comment:
aside, I wonder if we should do this after every gatherer. I recently got burned by it not starting at 0,0 in the images one